backend/src/calibration.py

```diff
@@ -40,8 +40,10 @@ def color_calibrate(packet):
     packet.unload_target()
 
     t = time.perf_counter()
-    res = minimize(_xyz, INIT_MOARR, (camsigs, LAB_REF), method='Powell')
-    print("Minimize: " + str(time.perf_counter() - t))
+    #res = minimize(_xyz, INIT_MOARR, (camsigs, LAB_REF), method='Powell')
+    res = fmin(_xyz, INIT_MOARR, (camsigs, LAB_REF))
```

Comment:
Why the function change?

Reply:
Those are the two I'm testing with. I just flipped between them.